pom.xml

```diff
@@ -33,6 +44,10 @@
 			<artifactId>postgresql</artifactId>
 			<scope>runtime</scope>
 		</dependency>
+		<!-- <dependency>
```

Comment:
Can we remove commented code if we don't need it?

Reply:
yeah we probably should